packages/bitget-wallet/yarn.lock

```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
This file needs to be removed, you can install dependencies in the `root` package.json NX will detect them if they are used in specific project/wallet package.

Reply:
Already removed yarn.lock